src/lib.rs

```diff
@@ -9,9 +9,11 @@ use tracing_subscriber::{
 };
 
 pub use axum_tracing_opentelemetry::opentelemetry_tracing_layer;
-
 pub mod propagation;
 
+#[cfg(feature = "integration_test")]
+pub mod test;
```

Comment:
`hyper`-related staff also looks useful. Perhaps we could add another feature `"hyper"` to expose some additional methods like `get_trace` and `get_headers`.

Reply:
yes, let's do in a follow up PR.